src/scss/base/_base.scss

```
@@ -0,0 +1,16 @@
+*{
```

```suggestion
* {
```
Sữa mấy lỗi tương tự